### DISCUSSION TITLE
How to training the chatgpt LLM model to generate the code which i wanted?

### DISCUSSION THREAD

**Discussion by muzixiaosanshi** [microsoft/PromptCraft-Robotics#23] [General]
hello,everyone,do you know how to training gpt3.5-turbo and let it generate the coding which we want?

**Comment by muzixiaosanshi** [comment]
I baught a robot which you use in your experiment(brand is the same mycobot), I can communicate with chatgpt to let it generate the coding i want .But i only do it by talk with chatgpt and it have tokens limit. Do you know how to training the model?

**Comment by Xj1567589354** [comment]
what's the problem? If you want ChatGPT to generate the code you want, you must have a conversation with it. How can it know what code you want to generate if you talk to it?
